pdata/ptrace/ptraceotlp/generated_exportpartialsuccess.go

```diff
@@ -45,6 +45,14 @@ func (ms ExportPartialSuccess) MoveTo(dest ExportPartialSuccess) {
 	*ms.orig = otlpcollectortrace.ExportTracePartialSuccess{}
 }
 
+func (ms ExportPartialSuccess) getOrig() *otlpcollectortrace.ExportTracePartialSuccess {
+	return ms.orig
+}
+
+func (ms ExportPartialSuccess) getState() *internal.State {
+	return ms.state
+}
```

Comment:
There are a bunch of changes like this which seem unrelated to entities?

Reply:
They are semi-related. Just helper shortcuts to make Message->Slice work in the pcommon module (it wasn't supported before). I'll clean this up, just didn't want to block further prototyping.

Reply:
done